ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -169,6 +170,12 @@ sFlow is disabled.
 | --------- | --------------- | -----------| --------- |
 | Ethernet16 | 111-112 | 110 | out |
 
+##### Transceiver Frequency
+
+| Interface | Transceiver Frequency |
+| --------- | --------------------- |
+| Ethernet67 | 190050 |
+
```

Comment:
Instead of a whole new table just for frequency, lets make the table for Transceiver settings and then have keep the frequency as a column. Also please add some unit in the value.

Reply:
And if I could ask you to include the media override option in the table as well it would be good. Thanks!